supersim.go

```diff
@@ -27,8 +27,8 @@ var genesisL2JSON []byte
 var DefaultConfig = Config{
 	l1Chain: anvil.Config{ChainId: 1, Port: 8545, Genesis: genesisL1JSON},
```

Comment:
nit: I think we should also set the port here to 0, since we cannot guarantee that 8545 is available when running tests.